Dockerfile

```diff
@@ -55,7 +55,7 @@ RUN apt-get update && \
     curl -sSL http://neuro.debian.net/lists/trusty.us-ca.full >> /etc/apt/sources.list.d/neurodebian.sources.list && \
     apt-key adv --recv-keys --keyserver hkp://pgp.mit.edu:80 0xA5D32F012649A5A9 && \
     apt-get update && \
-    apt-get install -y fsl-core=5.0.9-3~nd14.04+1 && \
+    apt-get install -y fsl-core=5.0.9-1~nd+1+nd16.04+1 && \
```

Comment:
are those changes still necessary applicable to this PR? (applies to `connectome-workbench` as well)